src/Compatibility/Core/src/iOS/Renderers/BoxRenderer.cs

```diff
@@ -98,7 +98,12 @@ protected override void SetBackground(Brush brush)
 				return;
 
 			if (Brush.IsNullOrEmpty(brush))
+				brush = Element.Background;
```

Comment:
Why is this? Should you get from the element still?

Reply:
I mainly did that because I was just expanding on what the code already was doing but applying it to the brush.

https://github.com/dotnet/maui/blob/main/src/Compatibility/Core/src/iOS/Renderers/BoxRenderer.cs#L101
The behavior was already to call `SetBackgroundColor(Element.BackgroundColor)`

So, I figured let's just try the brush one more time. Though my guess is that it'll probably always just be a null or empty brush